bool reset
float64[] action
---
float64[] robot_state